examples/chip-tool/templates/tests/partials/test_cluster.zapt

```diff
@@ -318,8 +318,17 @@ class {{filename}}Suite: public TestCommand
     {
         chip::app::StatusIB status(error);
         {{#if response.error}}
+          {{#if optional}}
+          if (status.mStatus == chip::Protocols::InteractionModel::Status::UnsupportedAttribute){
+            NextTest();
```

Comment:
Does this not need the "unless async" bit?

Reply:
No, this is not necessary, because async is only on subscriptions report portion which is success case only. It doesn't hurt to have so I will add it, just in case we add an async expected failure response.